### ISSUE TITLE
Installing `pg_stat_statements` and rails' tasks

### ISSUE THREAD

**Issue by bruno-** [pawurb/rails-pg-extras#37]
Hi,
first off thank you for creating and maintaining this awesome gem!

After I install `pg_stat_statements` with `RailsPgExtras.add_extensions` I get the following lines in `db/schema.rb`:

```ruby
  enable_extension "pg_stat_statements"
  # ...
  create_view "pg_stat_statements_info", sql_definition: <<-SQL
      SELECT pg_stat_statements_info.dealloc,
      pg_stat_statements_info.stats_reset
     FROM pg_stat_statements_info() pg_stat_statements_info(dealloc, stats_reset);
  SQL
``` 

Now if I want to drop and reset a DB I get the following error with these rails commands:

- `bin/rails db:drop db:prepare`
- `bin/rails db:drop db:create db:schema:load`

```shell
Dropped database 'main_development'
Dropped database 'main_test'
Created database 'main_development'
rails aborted!
ActiveRecord::StatementInvalid: PG::DuplicateTable: ERROR:  relation "pg_stat_statements_info" already exists
/Users/bruno/data/linkok/main/db/schema.rb:387:in `block in <main>'
/Users/bruno/data/linkok/main/db/schema.rb:13:in `<main>'

Caused by:
PG::DuplicateTable: ERROR:  relation "pg_stat_statements_info" already exists
/Users/bruno/data/linkok/main/db/schema.rb:387:in `block in <main>'
/Users/bruno/data/linkok/main/db/schema.rb:13:in `<main>'
Tasks: TOP => db:schema:load
(See full trace by running task with --trace)
```

In either case, I think the problem is:
- Rails loads the schema from `db/schema.rb`
- It executes the line `enable_extension "pg_stat_statements"` and adds the extension which automatically creates the `pg_stat_statements_info` view.
- Now schema comes to the line with `create_view "pg_stat_statements_info", sql_definition ...`. This fails because the view already exists.

This effectively makes `db:prepare` task useless. The workaround is to use `db:create db:migrate db:seed`.

Do you have any suggestions how to handle this more elegantly? Thanks a lot!

**Comment by bruno-** [comment]
Ok, I found a better solution from this source: https://www.dixis.com/posts/155

TL;DR: just create a rails initializer with this:

```ruby
ActiveRecord::SchemaDumper.ignore_tables = [
  "pg_stat_statements_info"
]
```

Now the `pg_stat_statements_info` will be ignored from the `schema.rb` perspective and will not be added to it. All rails tasks `db:schema:load` and `db:prepare` now work fine.

**Comment by bruno-** [comment]
I think this issue can be closed. Maybe we can think about adding the instructions for this to the readme.

**Comment by pawurb** [comment]
Thanks, I'll double-check what's up and consider adding this info.
